internal/pkg/template/templates/addons/aurora/env/serverlessv2.yml

```diff
@@ -0,0 +1,178 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+  # Customize your Aurora Serverless cluster by setting the default value of the following parameters.
+  {{logicalIDSafe .ClusterName}}DBName:
+    Type: String
+    Description: The name of the initial database to be created in the Aurora Serverless v2 cluster.
+    Default: {{.InitialDBName}}
+    # Cannot have special characters
+    # Naming constraints: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html#RDS_Limits.Constraints
+  VPCID:
+    Type: String
+    Description: The ID of the VPC in which to create the Aurora Serverless v2 cluster.
+    Default: ""
+  PrivateSubnets:
+    Type: String
+    Description: The IDs of the private subnets in which to create the Aurora Serverless v2 cluster.
+    Default: """
+
+Mappings:
+  {{logicalIDSafe .ClusterName}}EnvScalingConfigurationMap: {{range $env := .Envs}}
+    {{$env}}:
+      "DBMinCapacity": 0.5 # AllowedValues: from 0.5 through 128
+      "DBMaxCapacity": 8   # AllowedValues: from 0.5 through 128
+    {{end}}
+    All:
+      "DBMinCapacity": 0.5 # AllowedValues: from 0.5 through 128
+      "DBMaxCapacity": 8   # AllowedValues: from 0.5 through 128
+
+Resources:
+  {{logicalIDSafe .ClusterName}}DBSubnetGroup:
+    Type: 'AWS::RDS::DBSubnetGroup'
+    Properties:
+      DBSubnetGroupDescription: Group of private subnets for Aurora Serverless v2 cluster.
+      SubnetIds:
+        !Split [',', !Ref PrivateSubnets]
+  
+  {{logicalIDSafe .ClusterName}}WorkloadSecurityGroup:
+    Metadata:
+      'aws:copilot:description': 'A security group for one or more workloads to access the Aurora Serverless v2 cluster {{logicalIDSafe .ClusterName}}'
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: !Sub 'The Security Group to access Aurora Serverless v2 cluster {{logicalIDSafe .ClusterName}}.'
+      VpcId: !Ref VPCID
+      Tags:
+        - Key: Name
+          Value: !Sub 'copilot-${App}-${Env}-{{ .ClusterName }}AuroraCluster'
```

Comment:
Why do we use `{{ .ClusterName }}` instead of `{{ .Name }}` to be consistent with the wkld addons?

Reply:
The field name is [`ClusterName`](https://github.com/aws/copilot-cli/blob/1e43932a3c6f2ff870bb25d743d53edee42a8756/internal/pkg/addon/storage.go#L163) instead of `Name`. We probably could rename the field to `Name` - but whether we name the field `ClusterName` or `Name` makes no difference to users. 

I *think* (but can't recall exactly) the intention behind `ClusterName` is to be clear that this would be the *exact* name of the cluster - the actual name of the cluster will be `ClusterName`, instead of `ClusterName-RANDOMSTRING`.

Reply:
I was trying to say `${Name}` but realized it is the wkld name. It reminds me tho since it is an addons that belongs to an env, maybe tagging it with  `copilot-${App}-${Env}-AuroraCluster` should be good enough?